numpy/ma/core.py

```diff
@@ -608,7 +608,9 @@ def common_fill_value(a, b):
 
 def filled(a, fill_value=None):
     """
-    Return input as an array with masked data replaced by a fill value.
+    Return a copy of `a`, with masked values replaced by a fill value.
+    However, if there are no masked values to fill, `a` will be returned
+    instead as an ndarray.
```

Comment:
Is it "`a`'s data will be returned" or "a copy of `a` will be returned"?

Reply:
Also note that this information shouldn't be part of the initial sentence, but rather the following paragraph.